### PR TITLE
Addition of hero component can be seen hero.html in src

### PR DESCRIPTION
**Pull request by 0sigma101**
Signed-off-by: 0sigma101 <pallav.gupta2021@vitstudent.ac.in>
my component can be seen in src as hero.html

### CODE REVIEW

File: src/hero.css [hunk 1]
```diff
@@ -0,0 +1,3 @@
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
```
Comment (AbhijithGanesh) [inline src/hero.css:3]: This part of the code is not required as it is already covered in [base.css](https://github.com/ComputerSocietyVITC/docHub/blob/master/css/base.css)

File: src/hero.html [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    <link href="/dist/output.css" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@100;200;300;400;500;600;700;800;900&family=Roboto&display=swap" rel="stylesheet">
+    <title>hero component</title>
+</head>
+<body class="bg-back">
+    <article class="inline-flex space-x-151">
+        <p class="w-737 h-151 text-tx font-bold font-text text-2xl text-justify py-20 px-51.7 mx-auto">IEEE Computer Society
+        VIT Chennai's Docs</p>
+        <p class="w-628.59 h-165.64 text-tx text-3xl text-justify space-y-737 px-51.7 py-51.7 mx-auto">The starting point for your tech journey, Get started and explore you favorite domain in tech. Whether you're just starting out or you're an experienced developer looking to take your skills to the next level, our documentation has something for <strong>everyone</strong>.</p>
+    </article>
```
Comment (AbhijithGanesh) [inline src/hero.html:16]: Only this bit of code is required if you re-wrote this in JSX

To learn what JSX is:
- [JSX from the React Team](https://reactjs.org/docs/introducing-jsx.html)
- [JSX - A comprehensive guide](https://facebook.github.io/jsx/)

File: tailwind.config.js [hunk 3]
```diff
@@ -41,6 +44,28 @@ module.exports = {
           logo: "#31CE9F",
         },
       },
+      fontSize: {
```
Comment (AbhijithGanesh) [inline tailwind.config.js:47]: These changes shouldn't be made. They're strongly not welcome. @porridgewithraisins @cool-pants

**Review by AbhijithGanesh — CHANGES_REQUESTED** [review]

**Comment by mainak-affinsys** [comment]
What even?

File: tailwind.config.js [hunk 4]
```diff
@@ -41,6 +44,28 @@ module.exports = {
...
+       }]
+     },
+     spacing: {
+      '737': '737px',
+      '151': '151px',
+      '628.59': '628.59px',
+      '165.64': '165.64px',
+      '51.7': '51.7px',
+      '20': '20px', 
+    },
```
Comment (0sigma101) [inline tailwind.config.js:68]: Do i need to change spacing too it actually does not interfere with the code

**Comment by cool-pants** [comment]
Why is the component named 'Hero', any specific reason for this change? What new things does it add/ change. Why is this required? Please specify all of this in the PR.

**Comment by 0sigma101** [comment]
> Why is the component named 'Hero', any specific reason for this change? What new things does it add/ change. Why is this required? Please specify all of this in the PR.

Hero is a synonym for website banners and that are used to put up advertisements but in our case we use it to put title

File: tailwind.config.js [hunk 5]
```diff
@@ -41,6 +41,17 @@ module.exports = {
...
+     },
+     spacing: {
+      '84':'84px',
+      '737': '737px',
+      '151': '151px',
+      '628.59': '628.59px',
+      '165.64': '165.64px',
+      '51.7': '51.7px',
+      '20': '20px', 
+    },
```
Comment (AbhijithGanesh) [inline tailwind.config.js:54]: We shouldn't be custom styling like this

**Review by AbhijithGanesh — CHANGES_REQUESTED** [review]

**Comment by 0sigma101** [comment]
Hey @AbhijithGanesh can please review my PR and the changes made.

**Comment by AbhijithGanesh** [comment]
@0sigma101 I'll review

**Comment by 0sigma101** [comment]
@AbhijithGanesh Please review. I the responsive is top notch now

File: tailwind.config.js [hunk 6]
```diff
@@ -41,6 +41,9 @@ module.exports = {
           logo: "#31CE9F",
         },
       },
+      fontSize: {
+     },
+     spacing: {},
     },
```
Comment (AbhijithGanesh) [inline tailwind.config.js:47]: LGTM

File: src/pages/index.tsx [hunk 7]
```diff
@@ -81,95 +83,8 @@ const IndexPage: React.FC<PageProps> = () => {
...
-                      Made with Gatsby 💖
-                    </section>
-                  </section>
-                </section>
-              </section>
-            </section>
-          </section>
+          <Hero />
+          <Flexi />
           <Footer />
```
Comment (AbhijithGanesh) [inline src/pages/index.tsx:88]: Re-strcturing

**Review by AbhijithGanesh — APPROVED** [review]
LGTM

**Review by AbhijithGanesh — APPROVED** [review]
LGTM -> Changes after footer rebase

**Pull request merged**